# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/tests/test_bug603550.html
# Commit: beeda5ef3712
# Full Hash: beeda5ef3712b73a19c7a4ce08a52ab4dd08b802
# Author: David P <daparks@mozilla.com>
# Date: 2024-07-04 16:09:52
# Regressor Bug: 1893119
# File Overlap Count: 1
# Description:
#   Bug 1893119: Part 20 - Move EndDragSession from nsIDragService to nsIDragSession r=gstoll,rkraesig,win-reviewers,geckoview-reviewers,m_kato
#   
#   This is a straightforward move.  It does take the liberty of breaking out an
#   EndDragSessionImpl method, which will be needed later.
#   
# ==============================================================================

diff -r 6c3ffc8c534c -r beeda5ef3712 layout/base/tests/test_bug603550.html
--- a/layout/base/tests/test_bug603550.html	Thu Jul 04 07:48:10 2024 +0000
+++ b/layout/base/tests/test_bug603550.html	Thu Jul 04 07:48:11 2024 +0000
@@ -71,7 +71,7 @@
     event.initDragEvent("drop", true, true, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null, null);
     fireEvent(element, event);
   } finally {
-    ds.endDragSession(false);
+    ds.getCurrentSession().endDragSession(false);
     ok(!ds.getCurrentSession(), "There shouldn't be a drag session anymore!");
   }
 }